From: Steven Chamberlain <steven@pyro.eu.org>
Description: properly support GStreamer on non-Linux platforms
 GStreamer is enabled when building on GNU/kFreeBSD and Hurd, but support
 for it was not enabled in the WTF component
Bug-Debian: http://bugs.debian.org/760727

---
 debian/patches/kfreebsd.diff |   11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 debian/patches/kfreebsd.diff

--- /dev/null
+++ b/debian/patches/kfreebsd.diff
@@ -0,0 +1,11 @@
+--- a/Source/WTF/WTF.pri
++++ b/Source/WTF/WTF.pri
+@@ -24,7 +24,7 @@
+     }
+ }
+ 
+-linux-*:contains(DEFINES, WTF_USE_GSTREAMER=1) {
++linux-*|glibc-*|hurd-*:contains(DEFINES, WTF_USE_GSTREAMER=1) {
+     DEFINES += ENABLE_GLIB_SUPPORT=1
+     PKGCONFIG += glib-2.0 gio-2.0
+ }
